### PR TITLE
Printing out test data predictions

### PR DESCRIPTION
**Pull request by arielah**
It's going to take a pick of digging through sklearn documentation to get the original names back, so in the interest of time I'm going to wait on that. But here are the probabilities paired with the annotated category as 'truth' column.

### CODE REVIEW

**Review by dhimmel — APPROVED** [review]
> It's going to take a pick of digging through sklearn documentation to get the original names back, so in the interest of time I'm going to wait on that.

fine to merge without, but out of curiosity, don't you know what the names are because you use that as an input at some point? I.e. can use the names from your input? Or is the ordering changed?

**Comment by arielah** [comment]
Fair question. The featurized names and labels are put into sklearn.model_selection.test_train_split() which does its own ordering, so I'm not sure how I'd map that back to the original order. I'm sure there's some way to do it, but it didn't seem necessary for our tight timeline. I think I'll make that an issue so I remember to do it later.

**Comment by dhimmel** [comment]
> The featurized names and labels are put into sklearn.model_selection.test_train_split() which does its own ordering

Ah makes sense. You may have to go one step up the abstraction heirarcy. From docs:

> Quick utility that wraps input validation and `next(ShuffleSplit().split(X, y))` and application to input data into a single call for splitting (and optionally subsampling) data in a oneliner.
